commander/test/bootstrapping/commands/transaction/create.spec.ts

```diff
@@ -590,6 +593,19 @@ describe('transaction:create command', () => {
 					});
 				});
 			});
+
+			describe(`transaction:create pos unlock 100000000 --params=${voteParams} --passphrase=${passphrase}`, () => {
```

Comment:
Test description mentions that `--params` flag is being used, while it correctly invokes command without `--params` flag.